src/Tribe/Assets.php

```diff
@@ -154,13 +174,17 @@ public function enqueue( $forcibly_enqueue = null ) {
 			} else {
 				wp_enqueue_style( $asset->slug );
 			}
+
+			$this->assets[ $key ]->already_enqueued = true;
```

Comment:
Ohh good catch @barryhughes we should use `$asset` it's a reference to the object.